<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgentMenu</class>
 <widget class="QMainWindow" name="AgentMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>855</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 250, 254);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="icon_name_widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	
	background-color: rgb(37, 150, 190);
}

QPushButton{
	color:white;
	height:30px;
	border:none;
	font: 10pt &quot;PT Root UI&quot;;
	border-radius:10px
}

QPushButton:checked{
	background-color:#f5fafe;
	color:#1f95ef;
	font-weight:bold
	
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="res.qrc">:/white/icons/white/home.svg</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_dashboard1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons2/icons/icon2/dashboard_white.png</normaloff>
             <normalon>:/icons2/icons/icon2/dashboard.png</normalon>:/icons2/icons/icon2/dashboard_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_manage">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/white/icons/white/search.svg</normaloff>
             <normalon>:/icons2/icons/icon2/iconizer-search.svg</normalon>:/white/icons/white/search.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_profile1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons2/icons/icon2/profile_white.png</normaloff>
             <normalon>:/icons2/icons/icon2/profile.png</normalon>:/icons2/icons/icon2/profile_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_favorites1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/white/icons/white/heart.svg</normaloff>
             <normalon>:/icons2/icons/icon2/iconizer-heart.svg</normalon>:/white/icons/white/heart.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_messages1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons2/icons/icon2/messages_white.png</normaloff>
             <normalon>:/icons2/icons/icon2/messages.png</normalon>:/icons2/icons/icon2/messages_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>246</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_signout1">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icons2/icons/icon2/log_out_white.png</normaloff>
           <normalon>:/icons2/icons/icon2/log_out.png</normalon>:/icons2/icons/icon2/log_out_white.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="main_menu" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="header_widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_nav_bar">
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icons2/icons/icon2/menu.png</normaloff>:/icons2/icons/icon2/menu.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>140</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>7</number>
            </property>
            <item>
             <widget class="LineEdit" name="LineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_search">
              <property name="styleSheet">
               <string notr="true">border:none</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/icons2/icons/icon2/search.png</normaloff>:/icons2/icons/icon2/search.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>162</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_user">
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icons2/icons/icon2/image.png</normaloff>:/icons2/icons/icon2/image.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="SlideAniStackedWidget" name="SlideAniStackedWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_dashborad">
          <widget class="TitleLabel" name="TitleLabel">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>70</y>
             <width>261</width>
             <height>101</height>
            </rect>
           </property>
           <property name="text">
            <string>Dashboard</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_manage">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>PT Root UI Bold</family>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>Property info</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <family>PT Root UI Bold</family>
                 <pointsize>10</pointsize>
                 <italic>false</italic>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>Welcome to property management page</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="FilledPushButton" name="btn_addProperty">
               <property name="text">
                <string>Add property</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="SearchLineEdit" name="SearchLineEdit">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>search user name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="TitleLabel" name="TitleLabel_2">
               <property name="text">
                <string>Total property</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="SmoothScrollArea" name="SmoothScrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>839</width>
                <height>618</height>
               </rect>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_profile">
          <layout class="QVBoxLayout" name="verticalLayout_6"/>
         </widget>
         <widget class="QWidget" name="page_favorites">
          <widget class="TitleLabel" name="TitleLabel_4">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>140</y>
             <width>123</width>
             <height>38</height>
            </rect>
           </property>
           <property name="text">
            <string>Favorites</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_messages">
          <widget class="TitleLabel" name="TitleLabel_5">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>190</y>
             <width>123</width>
             <height>38</height>
            </rect>
           </property>
           <property name="text">
            <string>Messages</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="icon_name_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(37, 150, 190);
	color:white;
}

QPushButton{
	color:white;
	text-align:left;
	height:30px;
	font: 10pt &quot;PT Root UI&quot;;
	border:none;
	padding-left:10px;
	border-top-left-radius:10px;
	border-bottom-left-radius:10px;
}

QPushButton:checked{
	background-color:#f5fafe;
	color:#1f95ef;
	font-weight:bold

}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="rightMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="res.qrc">:/white/icons/white/home.svg</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>PT Root UI Bold</family>
             <pointsize>14</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;PT Root UI Bold&quot;;</string>
           </property>
           <property name="text">
            <string>Real Estate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_dashboard2">
           <property name="text">
            <string>dashboard</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons2/icons/icon2/dashboard_white.png</normaloff>
             <normalon>:/icons2/icons/icon2/dashboard.png</normalon>:/icons2/icons/icon2/dashboard_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_manage2">
           <property name="text">
            <string>property Manage</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/white/icons/white/search.svg</normaloff>
             <normalon>:/icons2/icons/icon2/iconizer-search.svg</normalon>:/white/icons/white/search.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_profile2">
           <property name="text">
            <string>Profile</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons2/icons/icon2/profile_white.png</normaloff>
             <normalon>:/icons2/icons/icon2/profile.png</normalon>:/icons2/icons/icon2/profile_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_favorites2">
           <property name="text">
            <string>Favorites</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/white/icons/white/heart.svg</normaloff>
             <normalon>:/icons2/icons/icon2/iconizer-heart.svg</normalon>:/white/icons/white/heart.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_messages2">
           <property name="text">
            <string>Messages</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons2/icons/icon2/messages_white.png</normaloff>
             <normalon>:/icons2/icons/icon2/messages.png</normalon>:/icons2/icons/icon2/messages_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>246</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_signout2">
         <property name="text">
          <string>Sign out</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icons2/icons/icon2/log_out_white.png</normaloff>
           <normalon>:/icons2/icons/icon2/log_out.png</normalon>:/icons2/icons/icon2/log_out_white.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>FilledPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgetspro</header>
  </customwidget>
  <customwidget>
   <class>SmoothScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SlideAniStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>qfluentwidgetspro</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SearchLineEdit</class>
   <extends>LineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_messages1</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_messages2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_favorites1</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_favorites2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_profile1</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_profile2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_manage</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_manage2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_dashboard1</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_dashboard2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_dashboard2</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_dashboard1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_manage2</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_manage</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_profile2</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_profile1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_favorites2</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_favorites1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_messages2</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_messages1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_nav_bar</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_name_widget_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_nav_bar</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_name_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_signout1</sender>
   <signal>toggled(bool)</signal>
   <receiver>AgentMenu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>667</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_signout2</sender>
   <signal>toggled(bool)</signal>
   <receiver>AgentMenu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>696</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
